assets/scss/theme/theme.scss

```diff
@@ -3,5 +3,5 @@
 }
 
 a:hover {
-  font-weight: bold;
+  text-decoration: underline;
 }
```

Comment:
Let's do this in a separate PR. I tried to do this in the initial Hinode migration, but there is weirdness with non-text links such as icons that I haven't figured out how to fix:

<img width="73" alt="image" src="https://github.com/user-attachments/assets/68dc4bef-3c38-40e7-9608-75434f0b4c52" />

Reply:
https://github.com/passkeydeveloper/passkeys.dev/issues/413

Reply:
Sure thing, I've reverted this.